src/main/java/org/jenkinsci/gradle/plugins/jpi2/ConfigurePrepareServerAction.java

```diff
@@ -69,10 +66,27 @@ public String transform(@NotNull String s) {
 
                         var jpiTask = dependencyProject.get().getTasks().findByName("jpi");
                         if (jpiTask != null) {
-                            sync.from(jpiTask.getOutputs().getFiles())
-                                    .into(projectRoot + "/work/plugins");
+                            sync.from(jpiTask.getOutputs().getFiles());
```

Comment:
If it's already modeled as an output, can this be shortened to from(jpiTask)?

Reply:
Thanks! Done.